lib/streamlit/hello/hello.py

```diff
@@ -20,6 +20,7 @@
     """
     Streamlit is an open-source app framework built specifically for
     machine learning and data science projects.
+
```

Comment:
This was one of the main things I wanted to fix: break the line below into its own paragraph.